source/renderer/app/stores/StakingStore.js

```diff
@@ -237,6 +240,18 @@ export default class StakingStore extends Store {
     this.getStakePoolsData(true);
   };
 
+  @action _getStakingInfoWasOpen = async () => {
+    const stakingInfoWasOpen = await this.api.localStorage.getStakingInfoWasOpen();
+    runInAction(() => {
+      this.stakingInfoWasOpen = stakingInfoWasOpen || false;
```

Comment:
@daniloprates isn't `stakingInfoWasOpen` already boolean, do you need `false` fallback?